virl2_client/models/users.py

```diff
@@ -36,8 +36,9 @@ class UserManagement:
         "user_id": "users/{username}/id",
     }
 
-    def __init__(self, session: httpx.Client) -> None:
+    def __init__(self, session: httpx.Client, is_rp_supported: bool = True) -> None:
         self._session = session
+        self._is_rp_supported = is_rp_supported
```

Comment:
wouldn't it be swell if the classes shared some context object, where such flags could be stored?